aep/general/0005/aep.md

```diff
@@ -0,0 +1,84 @@
+# Designing an API
+
+This AEP serves as a high-level guide on designing an API that adheres to the
+AEPs.
+
+## Process summary
+
+1. enumerate the user case would like your API to satisfy.
+1. identify resources and add them to your API.
+1. identify standard operations and add them to your API.
+1. identify custom operations and add them to your API.
+
+## Enumerate use cases
+
+The first step in designing an API is understanding precisely what you would
```

Comment:
Is this necessarily true?  Sometimes you might design an API and hope your users surprise you by building cool stuff with it.

Maybe "precisely which operations you would like your users to be able to perform"?

Reply:
Sure, I meant the latter - I kind of feel like "accomplish" and "perform" are synonymous here, but since you raised the comment I guess not. 

Applied your suggestion.